homeassistant/components/hangouts/hangouts_bot.py

```diff
@@ -214,10 +220,48 @@ def _on_disconnect(self):
                                                segment_type=hangouts_pb2.
                                                SEGMENT_TYPE_LINE_BREAK))
 
+        image_file = None
+        if data and data.get('image') is not None:
+            uri = data.get('image')
+            validate = urlparse(uri)
+            if validate.scheme:
+                try:
+                    websession = async_get_clientsession(self.hass)
+                    with async_timeout.timeout(5, loop=self.hass.loop):
+                        response = await websession.get(uri)
```

Comment:
That is the old style of timeout handling. Please use the new one with `async with websession.get(uri, timeout=5) as resp`